src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties

```diff
@@ -478,7 +478,8 @@ doclet.usage.author.description=\
     Include @author paragraphs
 
 doclet.usage.docfilessubdirs.description=\
-    Recursively copy doc-file subdirectories
+    Enables deep copying of `doc-files` directories. Subdirectories and all\n\
```

Comment:
Fixed in [e24bbd8](https://github.com/openjdk/jdk/pull/20618/commits/e24bbd8d6c1f55f2e43ad453703c26105d0b4e7a), thanks.

Reply:
@nizarbenalla have you checked that the help output is rendered correctly after you changed backticks to single quotes?

Reply:
Yes, I checked locally on my Mac machine and the output looks correct. ie
```
....
    -overview <file>
                  Read overview documentation from HTML file
    -serialwarn   Reports compile-time warnings for missing '@serial' tags
    --since <release>(,<release>)*
                  Document new and deprecated API in the specified releases
....
```